components/employees/add-employee-form/index.tsx

```diff
@@ -0,0 +1,453 @@
+                    {/* TODO: Role list will be added later */}
+                    {/* <SelectItem></SelectItem> */}
+                    <div>Not Found</div>
+                  </SelectContent>
+                </Select>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name="approvalFlowId"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>{approvalFlow?.label}</FormLabel>
+
+                <Select
+                  onValueChange={field.onChange}
+                  defaultValue={field?.value}
+                >
+                  <FormControl>
+                    <SelectTrigger
+                      className={`${field?.value === undefined ? "text-placeholder" : "text-textColor"}`}
+                    >
+                      <SelectValue placeholder={approvalFlow?.placeholder} />
+                    </SelectTrigger>
+                  </FormControl>
+                  <SelectContent>
+                    {/* TODO: Approval Flows will be added later */}
+                    {/* <SelectItem></SelectItem> */}
+                    <div>Not Found</div>
+                  </SelectContent>
+                </Select>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name="holidayCalender"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>{holidayCalender?.label}</FormLabel>
+
+                <Select
+                  onValueChange={field.onChange}
+                  defaultValue={field?.value}
+                >
+                  <FormControl>
+                    <SelectTrigger
+                      className={`${field?.value === undefined ? "text-placeholder" : "text-textColor"}`}
+                    >
+                      <SelectValue placeholder={holidayCalender?.placeholder} />
+                    </SelectTrigger>
+                  </FormControl>
+                  <SelectContent>
+                    {/* TODO: Holiday Calender will be added later */}
+                    {/* <SelectItem></SelectItem> */}
+                    <div>Not Found</div>
+                  </SelectContent>
+                </Select>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name="gender"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>{gender?.label}</FormLabel>
+
+                <Select
+                  onValueChange={field.onChange}
+                  defaultValue={field?.value}
+                >
+                  <FormControl>
+                    <SelectTrigger
+                      className={`${field?.value === undefined ? "text-placeholder" : "text-textColor"}`}
```

Comment:
here as well

Reply:
@imrankhan-stixor , this is not an input field so `placeholder:text-placeholder` will not work